README.md

```diff
@@ -5,6 +5,7 @@ LevelDOWN
 
 **A Low-level Node.js LevelDB binding**
 
+![Node version](https://img.shields.io/node/v/leveldown.svg)
 [![Build Status](https://secure.travis-ci.org/Level/leveldown.png)](http://travis-ci.org/Level/leveldown)
 [![AppVeyor build status](https://img.shields.io/appveyor/ci/Level/leveldown.svg)](https://ci.appveyor.com/project/Level/leveldown)
 [![dependencies](https://david-dm.org/Level/leveldown.svg)](https://david-dm.org/level/leveldown) [![Greenkeeper badge](https://badges.greenkeeper.io/Level/leveldown.svg)](https://greenkeeper.io/)
```

Comment:
Maybe we can remove dependencies badge now since we use greenkeeper?

Reply:
Does the greenkeeper badge have the same meaning? Currently it says "enabled". Which other states does it have?

Reply:
I think it only has that state but it will let us know when dependencies are out of date. We can keep the other badge too, no biggie.

Reply:
Ok. Yeah, I say keep David as a signal to users :)